news/2022-02-25-skinning-contest-tides-of-winter-results.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Skinning Contest: Tides of Winter - Results"
+date: 2022-02-25 12:00:00 +0000
+---
+
+As the Winter season gradually wraps up, the results of the chilliest skinning contest to date arrive at your doorstep warmer than ever!
+
+![](/wiki/shared/news/2022-02-25-skinning-contest-tides-of-winter-results/banner.jpg)
+
+No matter how well the titular tides of winter were represented throughout all **17** skins, your votes ultimately decided the most wintry of the bunch. 
+
+Bitter choices were made with so few votes to go around, yet you managed to throw your *snowballots* right back at us in numbers we never expected, with a total of **17674** frosty votes made away by **6946** unique users barreling into our front yard like an avalanche!
+
+Naturally, it took us a bit of time to count all the votes, make necessary preparations for the announcement and, you know, plough the snow on our yard. So without further ado:
+
+## 3rd Place - Azure Snowfall
+
+Capturing a total of **1697** hearts with an exquisite yet simplistic piece of art, [Azure Snowfall](https://osu.ppy.sh/community/forums/topics/1498492), [thetasigma](https://osu.ppy.sh/users/6234482) achieves the 3rd place as the contest comes to an end!
+
+Projecting the tides of winter in a minimalistic approach, Azure Snowfall, as its name suggests, culminates the flakes of snow and the beauty of the sky at night. Opting for shades of dark blue tones almost everywhere, the skin neatly and artistically accomodates both snow and sky, providing a stargazing-inspired experience featuring the bluest of mountain zeniths where flakes and stars meet.
+
+The song selection menu greets us with a texture reminiscent of the middle of Alps. The remainder holds an elegant yet minimalistic aura, exemplified in the pause and fail menus. Furthermore, the gameplay section from top to bottom is left fairly simple, which goes with the general theme fascinatingly well! If you don't believe my words, check what sort of a masterpiece Azure Snowfall really is!
+
+<iframe width="100%" height="400" src="https://www.youtube-nocookie.com/embed/Za27g2rzhOc" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Having snowy mountains and shiny stars meet at the horizon, thetasigma gets to achieve the 3rd place profile badge with their Azure Snowfall entry! Congratulations!
+
+![](/wiki/shared/news/2022-02-25-skinning-contest-tides-of-winter-results/azure.png)
+
+## 2nd Place - GanyuTeks:Futur Céleste
+
+**2006** voters chose none other than this tech-style masterpiece delicately crafted by the very hands of [BPChingu](https://osu.ppy.sh/users/4236855), [GanyuTeks:Futur Céleste](https://osu.ppy.sh/community/forums/topics/1489926), claiming 2nd place on the podium!
```

Comment:
the rhyming must continue~

also keep the order of skin, skinner

```suggestion
**2006** voters were brought a smile by none other than this masterpiece of delicate tech style, letting [GanyuTeks:Futur Céleste](https://osu.ppy.sh/community/forums/topics/1489926) by [BPChingu](https://osu.ppy.sh/users/4236855) claim 2nd place!
```

Reply:
> [...], letting GnayuTeks by BPChingu 2nd place!

idk if this is just me, but that sounds weird

Reply:
> letting [GanyuTeks:Futur Céleste](https://osu.ppy.sh/community/forums/topics/1489926) by [BPChingu](https://osu.ppy.sh/users/4236855) claim 2nd place!

did ya miss a word while reading? doesn't sound very off to me

Reply:
whether the *entry* or the *creator* claims the prize is a matter of perspective, and it's completely fair to say it was the *contest entry* that won, or the other

... just to cover my bases, in case that was what you meant